Gazel-Tutorial/src/Gazel.Tutorial.Module.ProductManagement/Service/ICartService.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Gazel.Tutorial.Module.ProductManagement.Service
+{
+    public interface ICartService
+    {
+        int Id { get; }
+        List<CartItem> CartProducts { get; }
```

Comment:
`CartItem` return edersen burasi CardProductIds olarak expose olur, `ICartItemInfo` gibi bir data interface'i yapman gerekir

bkz. [ITaskInfo](https://github.com/multinetinventiv/gazel-samples/blob/17947dee7c727c0173b3959253a547041825e4a2/to-do/src/Inventiv.Todo.Module.TaskManagement/Service/ITaskService.cs#L46)

Reply:
Cart artık CartItem listesi veya CartItem tutmuyor, 6. satırı silmeyi unutmuşum. Şimdi düzelttim.

Reply:
bi dk bunu fark etmemişim, ICartService'te property olmamalı, service interface'leri yalnızca method için

Reply:
yanlış yazmışım ICartInfo olucaktı, şimdi düzelttim.